samples/coco/coco.py

```diff
@@ -428,13 +435,23 @@ def evaluate_coco(model, dataset, coco, eval_type="bbox", limit=0, image_ids=Non
                         metavar="<True|False>",
                         help='Automatically download and unzip MS-COCO files (default=False)',
                         type=bool)
+    parser.add_argument('--backbone', required=False,
+                        default="resnet50",
+                        metavar="<backbone>",
+                        help='Feature Pyramid Network backbone type')
+    parser.add_argument('--train_bn', required=False,
+                        default=False,
+                        metavar="<True|False>",
+                        help='Train batch norm layers (default=False)',
+                        type=bool)
```

Comment:
I don't see the need for a specific command line parameter to control BN training. All the other training parameters are set in Config, so I think this belongs there as well.

Reply:
Cool I'll remove it